app/assets/javascripts/oxalis/model/helpers/nml_helpers.js

```diff
@@ -2,14 +2,34 @@
 
 import _ from "lodash";
 import { getPosition, getRotation } from "oxalis/model/accessors/flycam_accessor";
+import messages from "messages";
+import Saxophone from "@scalableminds/saxophone";
+import Store from "oxalis/store";
+import Date from "libs/date";
+import { getNodeToEdgesMap } from "oxalis/model/accessors/skeletontracing_accessor";
 import type {
   OxalisState,
   SkeletonTracingType,
   NodeMapType,
   EdgeType,
   TreeType,
+  TreeMapType,
+  TemporaryMutableTreeType,
+  TemporaryMutableTreeMapType,
 } from "oxalis/store";
 
+// NML Defaults
+const DEFAULT_TIME = 0;
+const DEFAULT_COLOR = [1, 0, 0];
+const DEFAULT_VIEWPORT = 0;
+const DEFAULT_RESOLUTION = 0;
+const DEFAULT_BITDEPTH = 0;
+const DEFAULT_INTERPOLATION = false;
+const DEFAULT_TIMESTAMP = 0;
```

Comment:
What is the difference between `DEFAULT_TIME` and `DEFAULT_TIMESTAMP`?

Reply:
`timestamp` is a per-node property, `time` is a global property of the NML. I don't know, whether they have different semantics or just different names.